gpu/tasks/nvidia_ctoolkit.yml

```diff
@@ -0,0 +1,55 @@
+---
+- name: Add NVIDIA GPG key
+  ansible.builtin.apt_key:
+    url: https://nvidia.github.io/libnvidia-container/gpgkey
+    state: present
+
+- name: Add NVIDIA repository
```

Comment:
I can see in gpu/tasks/main.yml - the Nvidia repo gets added only after the "nvidia_update" playbook. But it should be added before and be a prerequisite.